test/unit/specs/components/staking/PageBond.spec.js

```diff
@@ -37,19 +36,20 @@ describe("PageBond", () => {
           country: "Canada",
           moniker: "someOtherValidator"
         })
-        store.commit("addToCart", {
-          id: "pubkeyZ",
-          pub_key: {
-            type: "ed25519",
-            data: "pubkeyZ"
-          },
-          voting_power: 20000,
-          shares: 75000,
-          description: "descriptionZ",
-          country: "Chile",
-          moniker: "aChileanValidator",
-          revoked: true
-        })
+
+        let candidates = await node.candidates()
```

Comment:
candidates are directly exported from `lcdClientMock` as `validators`, no need to query them like this. we can leave it like this, though.